gradle.properties

```diff
@@ -1,4 +1,5 @@
 org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.configuration-cache=false
-org.gradle.jvmargs=-Xmx3500m -Xms1500m
\ No newline at end of file
+org.gradle.jvmargs=-Xmx3500m -Xms1500m
+kotlin.daemon.useFallbackStrategy=true
```

Comment:
På sida du linka til: 
However, a silent fallback to another strategy can consume a lot of system resources or lead to non-deterministic builds. Read more about this in this [YouTrack issue](https://youtrack.jetbrains.com/issue/KT-48843/Add-ability-to-disable-Kotlin-daemon-fallback-strategy).

Har du sett noe på YoutTrack-issuen? Litt vanskelig å si hvor stor risikoen er for non-deterministic builds, kanskje? Og brukes denne fallbacken (med risiko) også for bygget med Kotlin 1.x?

Reply:
Ser at moglegheita til å skru av fallback kom i 1.8, og iallfall da var defaultverdien true: https://kotlinlang.org/docs/whatsnew18.html#ability-to-disable-the-kotlin-daemon-fallback-strategy

Så trur dette er standardverdien også framleis, og at dette dermed ikkje er noko endring.

Men vi kan jo like godt unngå å definere verdi her, og dermed bruke det som ligg som standard i Kotlin. Har ikkje grunn til å tru at vi har verken meir eller mindre problem her enn andre, så da kan vi heller ha det i verktygkassa til det eventuelt blir eit problem